core/ecschema-metadata/src/UnitConversion/README.md

```diff
@@ -0,0 +1,66 @@
+<!-- prettier-ignore-start -->
```

Comment:
This doc doesn't belong here as such ... we should document how this all works but this is not the right format (it won't get included right in our documentation.